sky/engine/core/painting/Typeface.h

```diff
@@ -0,0 +1,32 @@
+// Copyright 2015 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef SKY_ENGINE_CORE_PAINTING_TYPEFACE_H_
+#define SKY_ENGINE_CORE_PAINTING_TYPEFACE_H_
+
+#include "sky/engine/tonic/dart_wrappable.h"
+#include "sky/engine/wtf/PassRefPtr.h"
+#include "sky/engine/wtf/RefCounted.h"
+#include "third_party/skia/include/core/SkTypeface.h"
+
+namespace blink {
+
+class Typeface : public RefCounted<Typeface>, public DartWrappable {
```

Comment:
I dont' think we're going to want more of these.  Instead we want to follow the implement-in-dart-and-copy-to-c++-when-used pattern like Rect and recently ColorFilter use.